src/main/webapp/dev/js/common/instructorFeedbackResults.js

```diff
@@ -147,11 +147,35 @@ function replaceButtonHtmlAndTooltipText(button, from, to) {
 function expandOrCollapsePanels(expandCollapseButton, panels) {
     const STRING_EXPAND = 'Expand';
     const STRING_COLLAPSE = 'Collapse';
+    const expandCollapseStudentsButton = 'a[id^="collapse-panels-button-section-"]';
+    const expandCollapseTeamButton = 'a[id^="collapse-panels-button-team-"]';
     // {@code panels} is not defined when {@code expandCollapseButton} is collapse panels button. We
     // need to define corresponding {@code targetPanels}.
-    const targetPanels = panels || $('div.panel-collapse');
 
     const isButtonInExpandMode = $(expandCollapseButton).html().trim().startsWith(STRING_EXPAND);
+
+    if (!panels && !isButtonInExpandMode) {
+        if ($(expandCollapseStudentsButton).length
+            && $(expandCollapseStudentsButton).html().trim().startsWith(STRING_EXPAND)) {
```

Comment:
I believe there were two cases related to three buttons( ExpandCollapseButton,   ExpandCollapseTeamButton, &  ExpandCollapseStudentButton.
1. When collapsing ExpandCollapseButton, make sure that all panels buttons under it ( ExpandCollapseTeamButton, &  ExpandCollapseStudentButton ) are in Collapse state .
2. When collapsing ExpandCollapseTeamButton, make sure ExpandCollapseStudentButton are in Collapse state.